lib/matplotlib/axes/_base.py

```diff
@@ -2866,35 +2884,24 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
             sb = None
         else:
             raise ValueError("%s is not a valid style value")
+        axis_map = {**self._get_axis_map(), 'both': self._get_axis_list()}
```

Comment:
`_get_axis_map` doesn't return a `Str: List[Axis]`, does it? It looks like it's `Str: Axis`.

Reply:
good catch, thanks, fixed.